packages/core/nuxt-theme-module/theme/components/InstagramFeed.vue

```diff
@@ -1,26 +1,31 @@
 <template>
-  <SfSection title-heading="Share Your Look" subtitle-heading="#YOURLOOK" class="section">
+  <SfSection title-heading="Share Your Look" subtitle-heading="#YOURLOOK" class="title">
     <div class="grid grid-images">
       <div class="grid__row">
         <div class="grid__col">
-          <SfImage src="/homepage/imageA.webp" :width="486" :height="486">katherina_trn</SfImage>
+          <SfImage v-if="isMobile" src="/homepage/imageAm.webp" :width="160" :height="160">katherina_trn</SfImage>
```

Comment:
can't we use media query in CSS instead? it avoids unnecessary JS inside of the component

Reply:
please see Filip's comment about hiding it with CSS